docs/source/class_basics.rst

```diff
@@ -77,6 +77,53 @@ to it explicitly using ``self``:
            a = self
            a.x = 1      # Error: 'x' not defined
 
+Class and instance variable annotations
+***************************************
+
+Mypy supports annotations for class and instance 
+variables in class bodies and methods. Use ``ClassVar`` to
+indicate to the static type checker that this variable 
```

Comment:
This a small nit, but the convention in the docs is say "the type checker" rather than "the static type checker".